luigi/contrib/external_program.py

```diff
@@ -132,6 +140,47 @@ def run(self):
                 tmp_stdout.close()
 
 
+class TrackingUrlContext(object):
```

Comment:
Can we make this private? `_TrackingUrlContext` - then you don't need to add documentation

Reply:
Thanks, done